src/crypto/PSASessionKeystore.cpp

```diff
@@ -192,5 +189,72 @@ void PSASessionKeystore::DestroyKey(HkdfKeyHandle & key)
     keyId = PSA_KEY_ID_NULL;
 }
 
+#if CHIP_CONFIG_ENABLE_ICD_CIP
+CHIP_ERROR PSASessionKeystore::PersistICDKey(Aes128KeyHandle & key)
+{
+    CHIP_ERROR err;
+    AesKeyAttributes attrs;
+    psa_key_id_t previousKeyId = key.As<psa_key_id_t>();
+    psa_key_attributes_t previousKeyAttrs;
+
+    psa_get_key_attributes(previousKeyId, &previousKeyAttrs);
```

Comment:
For 100% correctness, you should also call `psa_reset_key_attributes` when the attributes are not needed anymore. Consider extracting this to some `IsPersistentKey` helper.

Reply:
As discussed offline, I've found that we can easily modify `PersistICDKey` to accept `Symmetric128BitsKeyHandle` for both key types. I've updated the code.
Added `psa_reset_key_attributes` before returns as well